commit 7c5801f42e756f477b0d4a8862088e0fa2e5d600
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-05-01T00:32:53+00:00 (5 months ago)
    
    MANTA-4242 Muppet fails to change haproxy config after a cueball/zk shuffle
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index c54fb9f..f5d072c 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
         "once": "1.4.0",
         "vasync": "2.2.0",
         "verror": "1.10.0",
-        "zkstream": "0.11.3"
+        "zkstream": "0.11.4"
     },
     "devDependencies": {
         "nodeunit": "0.11.2",
